base/abstractarray.jl

```diff
@@ -905,8 +905,19 @@ julia> getindex(A, 2:4)
  3
  2
  4
+
+ julia> getindex(A, CartesianRange((2:4,)))
+3-element Array{Int64,1}:
+ 3
+ 2
+ 4
 ```
 """
+function getindex{T,N}(A::AbstractArray{T,N}, cr::CartesianRange{CartesianIndex{N}})
```

Comment:
This static parameter syntax is actually deprecated; instead, the signature should use `where`.

Reply:
Docstrings should be attached to the "general" method, not a specialized version.